test/data/psst-component-data/scripts/a/user.js

```diff
@@ -4,8 +4,17 @@
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
 (() => {
+  const getUserId = () => {
+    return "user1"
+  }
   return new Promise((resolve) => {
-    document.title = 'test';
-    resolve(true)
+    document.title = 'user-';
+    resolve({
+      "user": getUserId(),
+      "requests": [
+        {"name": "privacy setting #1"},
+        {"name": "privacy setting #2"},
+      ]
+    })
   })
-})();
+})();
```

Comment:
I recommend adding `fix,-indent-with-non-tab,trailing-space,cr-at-eol` to core.whitespace in your gitconfig

Reply:
also please fix the existing missing eol